src/components/slider/slider.ts

```diff
@@ -215,6 +241,80 @@ export class MdSlider implements AfterContentInit {
   }
 
   /**
+   * Calls the appropriate method for calculating the tick separation depending on what the tick
+   * interval is. If there is no tick interval or the interval is set to something other than a
+   * number or 'auto', nothing happens.
+   */
+  private _calculateTickSeparation() {
+    if (this.tickInterval == 'auto') {
+      this._calculateAutoTickSeparation();
+    } else if (typeof this.tickInterval == 'number') {
```

Comment:
This is needed because this function is called regardless of if `tick-interval` was set when creating a slider and ticks should only be drawn when `tick-interval` is explicitly set.